src/components/QuranReader/Loading.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Notes from './Notes/Notes';
+
+import Spinner, { SpinnerSize } from 'src/components/dls/Spinner/Spinner';
+
+interface Props {
+  isSideBarVisible: boolean;
+  containerClassName: string;
+  visibleSideBarClassName: string;
+}
+
+const Loading: React.FC<Props> = ({
+  isSideBarVisible,
+  containerClassName,
+  visibleSideBarClassName,
+}) => {
+  return (
+    <>
+      <div
+        className={classNames(containerClassName, {
+          [visibleSideBarClassName]: isSideBarVisible,
+        })}
+      >
+        <Spinner size={SpinnerSize.Large} isCentered />
+      </div>
+      <Notes />
```

Comment:
Is notes meant to be under loading?

Reply:
It was there from before. The idea is that if the Notes are open and the QuranReader is loading because let's say the user has just changed his Translations settings, the Notes should still be visible.